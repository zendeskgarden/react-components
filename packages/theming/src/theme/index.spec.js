/**
 * Copyright Zendesk, Inc.
 *
 * Use of this source code is governed under the Apache License, Version 2.0
 * found at http://www.apache.org/licenses/LICENSE-2.0.
 */

import DEFAULT_THEME from '.';

describe('DEFAULT_THEME', () => {
  it('matches snapshot', () => {
    expect(DEFAULT_THEME).toMatchSnapshot();
  });

  it('does not include `PALETTE.product`', () => {
    expect(DEFAULT_THEME.palette.product).toBeUndefined();
  });
});
