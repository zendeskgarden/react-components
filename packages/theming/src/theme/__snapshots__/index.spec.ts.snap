// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DEFAULT_THEME matches snapshot 1`] = `
Object {
  "borderRadii": Object {
    "md": "4px",
    "sm": "2px",
  },
  "borderStyles": Object {
    "solid": "solid",
  },
  "borderWidths": Object {
    "md": "3px",
    "sm": "1px",
  },
  "borders": Object {
    "md": "3px solid",
    "sm": "1px solid",
  },
  "colors": Object {
    "background": "#fff",
    "base": "light",
    "chromeHue": "kale",
    "dangerHue": "red",
    "foreground": "#2f3941",
    "neutralHue": "grey",
    "primaryHue": "blue",
    "successHue": "green",
    "warningHue": "yellow",
  },
  "components": Object {},
  "fontSizes": Object {
    "lg": "18px",
    "md": "14px",
    "sm": "12px",
    "xl": "22px",
    "xs": "10px",
    "xxl": "26px",
    "xxxl": "36px",
  },
  "fontWeights": Object {
    "black": 900,
    "bold": 700,
    "extrabold": 800,
    "extralight": 200,
    "light": 300,
    "medium": 500,
    "regular": 400,
    "semibold": 600,
    "thin": 100,
  },
  "fonts": Object {
    "mono": "SFMono-Regular,Consolas,\\"Liberation Mono\\",Menlo,Courier,monospace",
    "system": "system-ui,-apple-system,BlinkMacSystemFont,\\"Segoe UI\\",Roboto,Oxygen-Sans,Ubuntu,Cantarell,\\"Helvetica Neue\\",Arial,sans-serif",
  },
  "lineHeights": Object {
    "lg": "24px",
    "md": "20px",
    "sm": "16px",
    "xl": "28px",
    "xxl": "32px",
    "xxxl": "44px",
  },
  "palette": Object {
    "azure": Object {
      "400": "#3091ec",
      "600": "#1371d6",
      "M400": "#5f8dcf",
      "M600": "#3a70b2",
    },
    "black": "#000",
    "blue": Object {
      "100": "#edf7ff",
      "200": "#cee2f2",
      "300": "#adcce4",
      "400": "#5293c7",
      "500": "#337fbd",
      "600": "#1f73b7",
      "700": "#144a75",
      "800": "#0f3554",
    },
    "crimson": Object {
      "400": "#e34f32",
      "600": "#c72a1c",
      "M400": "#cc6c5b",
      "M600": "#b24a3c",
    },
    "fuschia": Object {
      "400": "#d653c2",
      "600": "#a81897",
      "M400": "#cf62a8",
      "M600": "#a8458c",
    },
    "green": Object {
      "100": "#edf8f4",
      "200": "#d1e8df",
      "300": "#aecfc2",
      "400": "#5eae91",
      "500": "#228f67",
      "600": "#038153",
      "700": "#186146",
      "800": "#0b3b29",
    },
    "grey": Object {
      "100": "#f8f9f9",
      "200": "#e9ebed",
      "300": "#d8dcde",
      "400": "#c2c8cc",
      "500": "#87929d",
      "600": "#68737d",
      "700": "#49545c",
      "800": "#2f3941",
    },
    "kale": Object {
      "100": "#f5fbfc",
      "200": "#c1d6d9",
      "300": "#819a9e",
      "400": "#56777a",
      "500": "#335d63",
      "600": "#04444d",
      "700": "#03363d",
      "800": "#012b30",
    },
    "lemon": Object {
      "400": "#ffd424",
      "600": "#ffbb10",
      "M400": "#e7a500",
      "M600": "#c38f00",
    },
    "lime": Object {
      "400": "#43b324",
      "600": "#2e8200",
      "M400": "#519e2d",
      "M600": "#47782c",
    },
    "mint": Object {
      "400": "#00a656",
      "600": "#058541",
      "M400": "#299c66",
      "M600": "#2e8057",
    },
    "orange": Object {
      "400": "#de701d",
      "600": "#bf5000",
      "M400": "#d4772c",
      "M600": "#b35827",
    },
    "pink": Object {
      "400": "#ec4d63",
      "600": "#d42054",
      "M400": "#d57287",
      "M600": "#b23a5d",
    },
    "purple": Object {
      "400": "#b552e2",
      "600": "#6a27b8",
      "M400": "#b072cc",
      "M600": "#9358b0",
    },
    "red": Object {
      "100": "#fff0f1",
      "200": "#f5d5d8",
      "300": "#f5b5ba",
      "400": "#e35b66",
      "500": "#d93f4c",
      "600": "#cc3340",
      "700": "#8c232c",
      "800": "#681219",
    },
    "royal": Object {
      "400": "#5d7df5",
      "600": "#3353e2",
      "M400": "#7986d8",
      "M600": "#4b61c3",
    },
    "teal": Object {
      "400": "#02a191",
      "600": "#028079",
      "M400": "#2d9e8f",
      "M600": "#3c7873",
    },
    "white": "#fff",
    "yellow": Object {
      "100": "#fff8ed",
      "200": "#fff0db",
      "300": "#fcdba9",
      "400": "#ffb648",
      "500": "#f5a133",
      "600": "#ed961c",
      "700": "#ad5e18",
      "800": "#703b15",
    },
  },
  "rtl": false,
  "shadowWidths": Object {
    "md": "3px",
    "sm": "2px",
  },
  "shadows": Object {
    "lg": [Function],
    "md": [Function],
    "sm": [Function],
  },
  "space": Object {
    "base": 4,
    "lg": "32px",
    "md": "20px",
    "sm": "12px",
    "xl": "40px",
    "xs": "8px",
    "xxl": "48px",
    "xxs": "4px",
  },
}
`;
